cob_teleop/config/teleop_logitech.yaml

```diff
@@ -0,0 +1,97 @@
+#cob4-2
```

Comment:
yaml's should go to `cob_robots/cob_hardware_config`
I guess this was just for testing :wink: